### DISCUSSION TITLE
No RichTextField documentation?

### DISCUSSION THREAD

**Discussion by mjmare** [wagtail/wagtail#10722] [General]
I'm a bit confused. [This page](https://docs.wagtail.org/en/stable/topics/pages.html#database-fields) mentions the RichTextFIeld. However there is no mention of this field in the Reference or the rest of the docs, except mention of a "legacy" Rich text field and a [page](https://docs.wagtail.org/en/stable/extending/rich_text_internals.html) on Rich text internals. 
Search is not a lot of help either. 

It is unclear to me what the status is of the RichTextFIeld and how to use it. I'd rather not use the StreamField for my use case.

Anyway, what I hoped to find in the docs is how to import HTML into a RichTextFIeld.

**Comment by zerolab** [comment]
@mjmare not sure if you've seen https://docs.wagtail.org/en/stable/advanced_topics/customisation/page_editing_interface.html#rich-text ? 🤔

**Reply by mjmare** [reply]
Hmm, no. It is not findable through search and for docs on fields I had not looked in "customising Wagtail"... ;-)
